index.js

```diff
@@ -66,7 +72,7 @@ async function run() {
 			releaseBody.push(`\n[\`${range}\`](https://github.com/${owner}/${repo}/compare/${range})`);
 		}
 
-		const octokit = getOctokit(process.env.RELEASE_TOKEN);
+		const octokit = getOctokit(core.getInput('token'));
```

Comment:
You can use this package to automatically pickup the token: https://github.com/octokit/auth-action.js

Reply:
That sounds cool, do you want me to add this to #22?

Reply:
Doesn’t sound related 😃